.ci/E2E-tests/execute.sh

```
@@ -3,24 +3,21 @@
 CONFIGURATION=$1
 TEST_FRAMEWORK=$2
 TEST_PATHS=$3  # Optional: space-separated list of test paths to run (passed through as-is, e.g., "e2e/exam/ e2e/atlas/")
-DB="mysql"
+DB="postgres"
```

Shellcheck (0.11.0)

[warning] DB appears unused. Verify use (or export if used externally).

(SC2034)